cognite/client/_api/iam.py

```diff
@@ -532,6 +532,7 @@ class SessionsAPI(APIClient):
     def __init__(self, config: ClientConfig, api_version: str | None, cognite_client: CogniteClient) -> None:
         super().__init__(config, api_version, cognite_client)
         self._LIST_LIMIT = 100
+        self._DELETE_LIMIT = 100
```

Comment:
Let's add a comment here saying there isn't really an API limit, but a self-inflicted one due to no support for REALLY large payloads.